src/status_im/integration_test/standard_auth_test.cljs

```diff
@@ -0,0 +1,100 @@
+(ns status-im.integration-test.standard-auth-test
+  (:require
+    [cljs.test :refer [deftest testing is use-fixtures]]
+    [day8.re-frame.test :as rf-test]
+    re-frame.core
+    [test-helpers.integration :as h]
+    [utils.re-frame :as rf]))
+
+(defn fixture-re-frame
+  []
+  (let [restore-re-frame (atom nil)]
+    {:before #(reset! restore-re-frame (re-frame.core/make-restore-fn))
```

Comment:
I don't understand why this fixture is needed because as far as I checked the implementation of `run-test-async`, it already sets up a restore point.

Reply:
you're right, wasn't aware of this 👍

Reply:
removed in [87498ce](https://github.com/status-im/status-mobile/pull/18756/commits/87498ce4ac747a25cc9db444847a2804c7a0492a)